man/demi_wait.md

```diff
@@ -33,10 +38,17 @@ the calling thread to block (spin) until the timeout `abstime` expires.
 specified by the list of queue tokens `qts` and it has a length of `num_qts`. This system call may cause the calling
 thread to block (spin) indefinitely.
 
+`demi_timedwait_any()` waits for the first asynchronous I/O operation in a set to complete or for the expiration of a
+timeout, whichever happens first. The set of I/O operations is specified by the list of queue tokens `qts` and it has a
+length of `num_qts`.  The `abstime` parameter specifies an absolute timeout in seconds and nanoseconds since the
+Epoch. If the I/O operation has already completed when `demi_timedwait()` is called, then this system call never fails
+with a timeout error, regardless of the value of `abstime`. This system call may cause the calling thread to block
+(spin) until the timeout `abstime` expires.
+
 When `demi_wait()` and `demi_timedwait()` successfully completes, the structure pointed to by `qr_out` is filled in with
-the result value of the I/O operation that has completed. The `demi_wait_any()` system call behaves similarly, but it
-additionally sets `ready_offset` to indicate the index of that I/O operation in the list of queue tokens `qts` that has
-completed.
+the result value of the I/O operation that has completed. The `demi_wait_any()` and `demi_timedwait_any` system calls
+behave similarly, but it additionally sets `ready_offset` to indicate the index of that I/O operation in the list of
```

Comment:
I believe this "it" should be a "they", if I'm understanding the semantics you intend correctly.

Reply:
Thanks for noting this. It is fixed now.